gitlab_runner/check.py

```diff
@@ -0,0 +1,116 @@
+# stdlib
+import urlparse
+
+# 3rd party
+import requests
+
+from checks import CheckException
+from checks.prometheus_check import PrometheusCheck
+from config import _is_affirmative
+from util import headers
+
+class GitlabRunnerCheck(PrometheusCheck):
+
+    EVENT_TYPE = SOURCE_TYPE_NAME = NAMESPACE = 'gitlab_runner'
+    SERVICE_CHECK_NAME = 'gitlab_runner.can_connect'
+
+    DEFAULT_CONNECT_TIMEOUT = 5
+    DEFAULT_RECEIVE_TIMEOUT = 15
+
+    """
+    Collect Gitlab Runner metrics from Prometheus and validates that the connectivity with Gitlab
+    """
+    def __init__(self, name, init_config, agentConfig, instances=None):
+        super(GitlabRunnerCheck, self).__init__(name, init_config, agentConfig, instances)
+        # Mapping from Prometheus metrics names to Datadog ones
+        # For now it's a 1:1 mapping
+        # TODO: mark some metrics as rate
+        allowed_metrics = init_config.get('allowed_metrics')
+
+        if not allowed_metrics:
+            raise CheckException("At least one metric must be whitelisted in `allowed_metrics`.")
+
+        self.metrics_mapper = dict(zip(allowed_metrics, allowed_metrics))
+
+    def check(self, instance):
+        #### Metrics collection
+        endpoint = instance.get('prometheus_endpoint')
+        if endpoint is None:
+            raise CheckException("Unable to find prometheus_endpoint in config file.")
+
+        # By default we send the buckets
+        send_buckets = instance.get('send_histograms_buckets', True)
+        if send_buckets is not None and str(send_buckets).lower() == 'false':
```

Comment:
you can use `_is_affirmative` from `config` module:
```
send_buckets = _is_affirmative(instance.get('send_histograms_buckets', True))
```

Reply:
I'm also using it at line 68 🤦‍♂️